config.yml

```diff
@@ -1,26 +1,32 @@
 language: en
 pipeline:
-- name: WhitespaceTokenizer
-- name: CRFEntityExtractor
-- name: EntitySynonymMapper
-- name: CountVectorsFeaturizer
-  token_pattern: (?u)\b\w+\b
-- name: EmbeddingIntentClassifier
-- name: DucklingHTTPExtractor
-  url: http://localhost:8000
-  dimensions:
-  - amount-of-money
-  - time
-  - number
-- name: "SpacyNLP"
-  model: "en_core_web_md"
-- name: "SpacyEntityExtractor"
-  dimensions: ["PERSON"]
+  - name: ConveRTTokenizer
+  - name: ConveRTFeaturizer
```

Comment:
i would switch back to the WhitespaceTokenizer - it's more lightweight

Reply:
That would also require getting rid of the ConveRTFeaturizer though right? since it requires that tokenizer?

Reply:
yeah that's what i'm saying, just switch to the pipeline we had before

Reply:
Ok sure. It should still be DIET instead of  EmbeddingIntentClassifier though right?

Reply:
yes, like this basically:
```
language: "en"

pipeline:
  - name: WhitespaceTokenizer
  - name: RegexFeaturizer
  - name: LexicalSyntacticFeaturizer
  - name: CountVectorsFeaturizer
  - name: CountVectorsFeaturizer
    analyzer: "char_wb"
    min_ngram: 1
    max_ngram: 4
  - name: DIETClassifier
  - name: EntitySynonymMapper
  - name: ResponseSelector
```
reference: https://rasa.com/docs/rasa/nlu/choosing-a-pipeline/#the-short-answer